packages/vavite/tsup.config.bundled_v10kr34fv7.mjs

```diff
@@ -0,0 +1,24 @@
+// tsup.config.ts
```

Comment:
I"m not a maintainer/contributor (sorry), but I'd suspect this file should be deleted from the diff